src/cloudmask.jl

```diff
@@ -1,3 +1,51 @@
+function convert_to_255_matrix(img)
+    img_clamped = clamp.(img, 0.0, 1.0)
+    return round.(Int, img_clamped * 255)
+end
+
+function _get_masks(
+    ref_image::Matrix{RGB{Float64}};
+    prelim_threshold::Float64=Float64(110 / 255),
+    band_7_threshold::Float64=Float64(200 / 255),
+    band_2_threshold::Float64=Float64(190 / 255),
+    ratio_lower::Float64=0.0,
+    ratio_upper::Float64=0.75,
+    use_uint8::Bool=false,
+)::Tuple{BitMatrix,BitMatrix}
+
+    ref_view = channelview(ref_image)
+    ref_image_b7 = @view ref_view[1, :, :]
+    if use_uint8
+        ref_image_b7 = convert_to_255_matrix(ref_image_b7)
+    end
+
+    clouds_view = ref_image_b7 .> prelim_threshold
+    mask_b7 = ref_image_b7 .< band_7_threshold
+    mask_b2 = @view(ref_view[2, :, :])
+    if use_uint8
+        mask_b2 = convert_to_255_matrix(mask_b2)
+    end
+    mask_b2 = mask_b2 .> band_2_threshold
+
+    # First find all the pixels that meet threshold logic in band 7 (channel 1) and band 2 (channel 2)
+    # Masking clouds and discriminating cloud-ice
+
+    mask_b7b2 = mask_b7 .&& mask_b2
+
+    # Next find pixels that meet both thresholds and mask them from band 7 (channel 1) and band 2 (channel 2)
+    b7_masked = mask_b7b2 .* ref_image_b7
+
+    _b2 = @view(ref_view[2, :, :])
+    b2_masked = use_uint8 ? convert_to_255_matrix(_b2) : _b2
+    b2_masked = mask_b7b2 .* b2_masked
+
+    cloud_ice = Float64.(b7_masked) ./ Float64.(b2_masked)
+    mask_cloud_ice = @. cloud_ice >= ratio_lower && cloud_ice < ratio_upper
+
+    return mask_cloud_ice, clouds_view
+end
+
+
 """
     create_cloudmask(falsecolor_image; prelim_threshold, band_7_threshold, band_2_threshold, ratio_lower, ratio_upper)
 
```

Comment:
```suggestion
- `ref_image`: corrected reflectance false color image - bands [7,2,1]
```

Will this ever not be a falsecolor image? I think we deliberately called the var falsecolor to avoid confusion with which image was used for cloudmasking. Although, it should be obvious from the description and we do use `ref_image` throughout the functions.

Also add:
```
- `use_uint8`: true or false determines whether to convert to a 255 matrix rather than float
```